src/shell/Shell.cpp

```diff
@@ -645,8 +641,13 @@ export class Shell final: public CoreVM::Runtime
     std::unique_ptr<CoreVM::Program> _currentProgram;
     CoreVM::Runner::Globals _globals;
 
+#if defined(ENDO_TRACE)
```

Comment:
I think that should be `ENDO_TRACE_VM` then. What do you think? Reason is, I was confused about `ENDO_TRACE`'s use, when ._PARSER and ._LEXER were pretty obvious to me.

On the other hand, what about having one trace preprocessor define, and then enable it fine grained at runtime, similar to logstore in Contour?

Reply:
Issue with runtime would be to enable this logs for the tests